### PR TITLE
Limit editing of controls inactive subarray

### PR DESCRIPTION
**Pull request by mmphego**
This PR does the following:
- Limits editing of controls on Subarray Display only to subarray state "inactive".

JIRA: [MT-1740](https://skaafrica.atlassian.net/browse/MT-1740)

 ## Screenshot

image

image
image

image
image
image

### CODE REVIEW

File: app/scheduler/templates/subarray-config-container.html [hunk 1]
```diff
@@ -20,18 +20,18 @@
         </md-menu-content>
     </md-menu>
     <md-menu>
-        <span ng-disabled="!$root.expertOrLO" title="Selected Band" ng-click="$root.expertOrLO && $mdMenu.open($event)"
+        <span ng-disabled="!$root.expertOrLO" title="Selected Band" ng-click="$root.expertOrLO && $mdMenu.open($event) "
```
Comment (mmphego) [inline app/scheduler/templates/subarray-config-container.html:23]:
```suggestion
        <span ng-disabled="!$root.expertOrLO" title="Selected Band" ng-click="$root.expertOrLO && $mdMenu.open($event)"```

File: app/scheduler/templates/subarray-config-container.html [hunk 2]
```diff
@@ -49,14 +49,14 @@
...
-                <md-button ng-click="parent.vm.setProduct(product)">
+                <md-button ng-disabled="vm.subarray.state !== 'inactive'"  ng-click="parent.vm.setProduct(product)">
                     {{product}}
                 </md-button>
             </md-menu-item>
         </md-menu-content>
     </md-menu>
     <md-menu>
-        <span ng-disabled="!$root.expertOrLO" title="Selected On-sky Centre Frequency" ng-click="$root.expertOrLO && $mdMenu.open($event)"
+        <span ng-disabled="!$root.expertOrLO" title="Selected On-sky Centre Frequency" ng-click="$root.expertOrLO && $mdMenu.open($event) "
```
Comment (mmphego) [inline app/scheduler/templates/subarray-config-container.html:59]:
```suggestion
        <span ng-disabled="!$root.expertOrLO" title="Selected On-sky Centre Frequency" ng-click="$root.expertOrLO && $mdMenu.open($event)"
```

**Review by bxaia — APPROVED** [review]

**Review by lanceWilliams — APPROVED** [review]
Looks ok.

**Review by amakhaba — APPROVED** [review]
LGTM

**Review by bngcebetsha — APPROVED** [review]
Looks good

**Pull request merged**